docs/source/tutorial/trainer.rst

```diff
@@ -197,45 +199,42 @@ To use these wide variety of tools for your training task, pass :class:`~chainer
+  :meth:`~chainer.training.extensions.snapshot_object`
+     :meth:`~chainer.training.extensions.snapshot` extension above saves the whole :class:`~chainer.training.Trainer` object. However, in some cases, it is tedious to retrieve only the inside of the model.
+     By using :meth:`~chainer.training.extensions.snapshot_object`, you can save the particular object (in the example above, the model wrapped by :class:`~chainer.links.Classifier`) as a separeted snapshot.
+     Taking the snapshot of :attr:`~chainer.links.Classifier.predictor` is enough to keep all the trained parameters, because
+     :class:`~chainer.links.Classifier` (which is a subclass of :class:`~chainer.Chain`) keeps the model as its :attr:`~chainer.links.Classifier.predictor` property, and all the parameters are under this property.
 
-:class:`~chainer.training.extensions.PrintReport`
-.................................................
+  :meth:`~chainer.training.extensions.dump_graph`
+     This extension saves the structure of the computational graph of the model.
+     The graph is saved in `Graphviz <http://www.graphviz.org/>`_ dot format under the output directory of the :class:`~chainer.training.Trainer`.
 
-It outputs the spcified values to the standard output.
+  :class:`~chainer.training.extensions.Evaluator`
+     :class:`~chainer.dataset.Iterator`\ s that use the evaluation dataset and the model object are required to use :class:`~chainer.training.extensions.Evaluator` extension.
+     It evaluates the model using the given dataset (typically it's a validation dataset) at the specified timing interval.
 
-:class:`~chainer.training.extensions.PlotReport`
-................................................
+  :class:`~chainer.training.extensions.PrintReport`
+     This extension outputs the spcified values to the standard output.
 
-:class:`~chainer.training.extensions.PlotReport` plots the values specified by its arguments saves it as a image file which has the same naem as the :attr:`~chainer.training.extensions.PlotReport.file_name` argument.
+  :class:`~chainer.training.extensions.PlotReport`
+     This extension plots the values specified by its arguments and saves it as a image file.
 
-----
 
-Each :class:`~chainer.training.Extension` class has different options and some extensions are not mentioned here. And one of other important feature is, for instance, by using the :attr:`~chainer.training.Extension.trigger` option, you can set individual timings to fire the :class:`~chainer.training.Extension`. To know more details of all extensions, please take a look at the official document: `Trainer extensions <reference/extensions.html>_`.
+This is not the exhaustive list of built-in extensions.
```

Comment:
This is not the exhaustive list -> This is not an exhaustive list

Reply:
Fixed!